### PR TITLE
Management group

### PR DESCRIPTION
**Pull request by haiyuazhang**
##### SUMMARY
new module azure_rm_managementgroup

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME

this module aimed to create/delete/update azure management group

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by Fred-sun** [comment]
ping to review！

File: lib/ansible/modules/cloud/azure/azure_rm_managementgroup.py [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+        if response:
+            self.results["id"] = response["id"]
+            self.results["type"] = response["type"]
+            self.results["name"] = response["name"]
+            self.results["properties"] = response["properties"]
+
+        return self.results
+
+    def create_update_resource(self):
+        # self.log('Creating / Updating the ManagementGroup instance {0}'.format(self.))
```
Comment (gavinfish) [inline lib/ansible/modules/cloud/azure/azure_rm_managementgroup.py:352]: Delete debug codes like this?

**Comment by Fred-sun** [comment]
@haiyuazhang close_me, instead by #67379

**Comment by gavinfish** [comment]
close_me

**Pull request closed without merging**